doc/model_persistence.rst

```diff
@@ -182,3 +270,69 @@ not help in production when performance is critical.
 To convert scikit-learn model to PMML you can use for example `sklearn2pmml
 <https://github.com/jpmml/sklearn2pmml>`_ distributed under the Affero GPLv3
 license.
+
+Choosing the right approach for model persistence
+-------------------------------------------------

```

Comment:
I have the sensation this whole section says almost the same as the `Summarizing the keypoints` section afterwards. I would rather keep the latter for easier reading, but maybe enrich the summary if needed, e.g.
```
* `ONNX`: It provides a uniform format for persisting any machine learning
  or deep learning model (other than scikit-learn) and is useful
  for model inference. It can however, result in compatibility issues with
  different frameworks.
```